src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  margin-right: 4px;
+}
+
+.stars--0 :last-child,
+.stars--1 :last-child,
+.stars--2 :last-child,
+.stars--3 :last-child,
+.stars--4 :last-child,
+.stars--5 :last-child {
+  margin-right: 0;
```

Comment:
.start__star:last-child {
  margin-right: 0;
}

Should work as well but shorter